### DISCUSSION TITLE
(very opinionated) Suggest hostnames.

### DISCUSSION THREAD

**Discussion by jtagcat** [archlinux/archinstall#354] [Ideas]
„There are 2 hurtles — caching and naming.“

https://web.archive.org/web/20090918202746/http://tothink.com/mnemonic/wordlist.html

**Comment by dylanmtaylor** [comment]
Is this a suggestion to have hostnames generated from a wordlist? It could be interesting, but I'm not sure I'd want to put that in. Might be amusing to do a [verb] [animal] type name like Ubuntu as the hostname. Just have to watch out for something like "Suggested hostname: jumping-jackass" lol.
